pegasus/routes/v2_section_routes.rb

```diff
@@ -13,6 +14,14 @@
   redirect "/v2/sections/#{section_id}"#, 201 #BUGBUG: JQuery is barfing on the 201
 end
 
+# Get the set of sections that the current user is enrolled in.
+get '/v2/sections/membership' do JSON.pretty_generate(:duid => dashboard_user_id)
```

Comment:
[API doc here.](http://www.ruby-doc.org/stdlib-1.9.3/libdoc/json/rdoc/JSON.html#method-i-pretty_generate)  It's what most of the other endpoints in this file use - I think it just puts additional line breaks and whitespace into the JSON output, which makes it read nicely when you hit it in the browser.  We could probably just use `JSON.generate` at cut a few bytes out of each request.